.github/workflows/build-toolchain.yml

```diff
@@ -306,24 +312,24 @@ jobs:
               "include": [
                 {
                   "arch": "amd64",
-                  "compiler_target": "x86_64-unknown-windows-msvc",
                   "os": "Windows",
                   "cc": "cl",
                   "cflags": "${{ steps.context.outputs.WINDOWS_CMAKE_C_FLAGS }}",
                   "cxx": "cl",
                   "cxxflags": "${{ steps.context.outputs.WINDOWS_CMAKE_CXX_FLAGS }}",
                   "swiftflags": "${{ steps.context.outputs.WINDOWS_CMAKE_Swift_FLAGS }}",
+                  "cmake_linker_flags": "-D CMAKE_EXE_LINKER_FLAGS=\"${{ steps.context.outputs.WINDOWS_CMAKE_EXE_LINKER_FLAGS }}\" -D CMAKE_SHARED_LINKER_FLAGS=\"${{ steps.context.outputs.WINDOWS_CMAKE_SHARED_LINKER_FLAGS }}\"",
```

Comment:
I wonder if we should split this up into `cmake_exe_linker_flags` and `cmake_shared_linker_flags`.